print_color_code.py

```diff
@@ -0,0 +1,10 @@
+import main
+
+print("Pair no.\t"+"Major Color\t"+"Minor Color")
```

Comment:
Good separation of reference manual in its own file. Enclose functionality in functions, or else we cannot control when it is called